tests/test_promptsdk_async.py

```
@@ -0,0 +1,207 @@
+        self.assertEqual(str(err), "API Error")
+
+
+def run_async_tests():
+    """
+    Helper function to run async tests
+    """
+    loop = asyncio.get_event_loop()
+    
+    # Create and run the test suite
+    suite = unittest.TestLoader().loadTestsFromTestCase(TestPromptSDKAsync)
+    runner = unittest.TextTestRunner()
```

Ruff (0.11.9)

Local variable `runner` is assigned to but never used

Remove assignment to unused variable `runner`

(F841)